firmware/application/apps/acars_app.cpp

```diff
@@ -31,73 +31,63 @@ using namespace acars;
 #include "string_format.hpp"
 #include "utility.hpp"
 
-void ACARSLogger::log_raw_data(const acars::Packet& packet, const uint32_t frequency) {
+void ACARSLogger::log_raw_data(const acars::Packet& packet,
+															 const uint32_t frequency) {
 	(void)frequency;
-	std::string entry { };	//= "Raw: F:" + to_string_dec_uint(frequency) + "Hz ";
+	std::string entry{};	//= "Raw: F:" + to_string_dec_uint(frequency) + "Hz ";
 	entry.reserve(256);
-	
+
 	// Raw hex dump of all the bytes
-	//for (size_t c = 0; c < packet.length(); c += 32)
+	// for (size_t c = 0; c < packet.length(); c += 32)
 	//	entry += to_string_hex(packet.read(c, 32), 8) + " ";
-	
+
 	for (size_t c = 0; c < 256; c += 32)
 		entry += to_string_bin(packet.read(c, 32), 32);
-	
+
 	log_file.write_entry(packet.received_at(), entry);
 }
 
 /*void ACARSLogger::log_decoded(
 	const acars::Packet& packet,
 	const std::string text) {
-	
+
 	log_file.write_entry(packet.timestamp(), text);
 }*/
 
 namespace ui {
 
 void ACARSAppView::update_freq(rf::Frequency f) {
 	set_target_frequency(f);
-	portapack::persistent_memory::set_tuned_frequency(f);	// Maybe not ?
+	portapack::persistent_memory::set_tuned_frequency(f);	 // Maybe not ?
 }
 
 ACARSAppView::ACARSAppView(NavigationView& nav) {
 	baseband::run_image(portapack::spi_flash::image_tag_acars);
 
-	add_children({
-		&rssi,
-		&channel,
-		&field_rf_amp,
-		&field_lna,
-		&field_vga,
-		&field_frequency,
-		&check_log,
-		&console
-	});
-	
+	add_children({&rssi, &channel, &field_rf_amp, &field_lna, &field_vga,
```

Comment:
isn't there a way to adjust this ?

Reply:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

look for binpack.

the ones that are really hard for formatters are the ones like Bernd’s frequency list alignment from the other day.

Reply:
Also AllowAllArgumentsOnNextLine

Reply:
I can do that tomorrow when I get back.

Reply:
Yeah because of the changes you guys want I'll have to abandoned this one and create a new one